frontend/src/app/rating/hr-side-menu/hr-side-menu.component.html

```diff
@@ -0,0 +1,32 @@
+<ion-menu side="end" contentId="hr-side-menu">
+  <ion-content>
```

Comment:
I suggest to reuse side-menu component may be we need some customisation to adopt changes. But the actions seems to be really the same.

Reply:
The problem is that the code related to HR is completely different compared to the one used for the "main" form. For instance, this side menu already uses Validators etc. So from my point of view, it is currently not trivial to make the side menus compatible with each other.

My suggestion: Leave it as it is and when the main form has been migrated to Validators etc, then we can think about merging it into one Component.